contracts/ethereum/MockETHLINKAggregator.go

```diff
@@ -4,6 +4,7 @@
 package ethereum
 
 import (
+	"errors"
```

Comment:
Was this file modified intentionally or did it take place when you ran **make compile_contracts.py?**

Reply:
Yes